### PR TITLE
[TSS-12] Test API burn

### PR DESCRIPTION
**Pull request by ayak002**
[TSS-12] test API burn

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -35,3 +35,13 @@ You can then exercise this API using the [postman collection](./postman/test-col
 This repository is automatically deployed when new changes are pushed to the `main` branch.
 
 If you need to deploy this yourself, simply `npm run build` then `npm run start`. This will run the `./dist/index.js` file on your node.js server. The main file is just an `express` server.
+
+## Test
+
+All the tests for the API "mint" and "burn" are in the `tests` folder at the root, the first test is testing that the client doesn't own a token, the second one, is testing that the operation to create a token went well, the third one is testing again if the client own or not the token, this time he should own a token because we just created one, and the last one it's testing that the token is correctly burned and so the client doesn't own a token anymore.
```
Comment (podefr) [inline INSTALL.md:41]: actually the tests are "self explanatory" so no need to describe them here (if you were the change the tests you'd also have to update this blurb, so you actually have duplication).
What's important to explain here is rather how to run the tests. The steps are:

1. start the flextesa standbox
2.start the node server 
3. run the tests

for each step, remember to add the relevant command line

thanks!

**Comment by ayak002** [comment]
okay ! thank you
________________________________

File: INSTALL.md [hunk 2]
```diff
@@ -35,3 +35,40 @@ You can then exercise this API using the [postman collection](./postman/test-col
...
+```sh
+./run compile-contract
+```
+
+after this you have run the deployed contract with this command line :
+
+```sh
+npm run deploy
+```
+(the node version has to be 18.17.0)
```
Comment (podefr) [inline INSTALL.md:58]:
```suggestion
(the node version has to be 18 or above)
```

File: INSTALL.md [hunk 3]
```diff
@@ -35,3 +35,40 @@ You can then exercise this API using the [postman collection](./postman/test-col
...
+
+after you deployed the contract you will receive an adress in the terminal that you need to copy and paste in your `.env`
+
+In case you don't know you run the node server with this command line :
+
+```sh
+npm start
+```
+
+The node server need to be on port 3000
```
Comment (podefr) [inline INSTALL.md:68]:
```suggestion
The node server needs to be on port 3000
```

**Review by podefr — APPROVED** [review]
LGTM

**Pull request merged**